react/features/desktop-picker/actionTypes.js

```diff
@@ -0,0 +1,20 @@
+import { Symbol } from '../base/react';
+
+/**
+ * Action to remove known DesktopCapturerSources.
+ *
+ * {
+ *     type: CLEAR_DESKTOP_SOURCES,
+ * }
+ */
+export const CLEAR_DESKTOP_SOURCES = Symbol('CLEAR_DESKTOP_SOURCES');
```

Comment:
You are absolutely correct; I meant reset. I screwed up while renaming clear to reset by missing several spots.